test/generate_ts_config.py

```diff
@@ -73,7 +73,7 @@ def update_training_service_config(args):
 
 def convert_command():
     '''convert command by platform'''
-    if sys.platform != 'win32':
+    if sys.platform != 'win32' or args.ts:
```

Comment:
why add this?

Reply:
this function is executed only if the platform if local, refactor logic,